Can't compile.

I have done this:
`
git clone https://github.com/ZigEmbeddedGroup/raspberrypi-rp2040.git
cd raspberrypi-rp2040/
`

Tried to build using this command:
`
zig build
`

Got this:
`
/home/alex/Documents/raspberrypi-rp2040/build.zig:7:30: error: unable to load '/home/alex/Documents/raspberrypi-rp2040/deps/microzig/build.zig': FileNotFound
pub const microzig = @import("deps/microzig/build.zig");
                             ^~~~~~~~~~~~~~~~~~~~~~~~~
`

Fixed this with:
`
gut clone https://github.com/ZigEmbeddedGroup/microzig.git
mv microzig/ deps/
`

After I try to build it again, I see this:
`
zig build-exe random Debug thumb-freestanding-none: error: the following command failed with 1 compilation errors:
/snap/zig/7686/zig build-exe /home/alex/Documents/raspberrypi-rp2040/deps/microzig/src/start.zig -fno-strip --cache-dir /home/alex/Documents/raspberrypi-rp2040/zig-cache --global-cache-dir /home/alex/.cache/zig --name random -fcompiler-rt -fsingle-threaded -target thumb-freestanding-none -mcpu cortex_m0plus --script /home/alex/Documents/raspberrypi-rp2040/rp2040.ld --mod config::/home/alex/Documents/raspberrypi-rp2040/zig-cache/microzig/config-969498c921793a9e671f14070bafbe74.zig --mod app:microzig:/home/alex/Documents/raspberrypi-rp2040/examples/random.zig --mod hal:microzig:/home/alex/Documents/raspberrypi-rp2040/src/hal.zig --mod cpu:microzig:/home/alex/Documents/raspberrypi-rp2040/deps/microzig/src/modules/cpus/cortex-m.zig --mod chip:microzig:/home/alex/Documents/raspberrypi-rp2040/src/chips/RP2040.zig --mod microzig:config,chip,cpu,hal,board:/home/alex/Documents/raspberrypi-rp2040/deps/microzig/src/microzig.zig --mod board:microzig:/home/alex/Documents/raspberrypi-rp2040/src/boards/raspberry_pi_pico.zig --deps app,microzig --listen=- 
Build Summary: 24/27 steps succeeded; 1 failed (disable with --summary none)
install transitive failure
└─ install random transitive failure
   └─ zig build-exe random Debug thumb-freestanding-none 1 errors
/snap/zig/7686/lib/std/math.zig:437:17: error: deprecated; use @min instead
pub const min = @compileError("deprecated; use @min instead");
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
referenced by:
    min: /snap/zig/7686/lib/std/rand/Ascon.zig:12:21
    fill: /snap/zig/7686/lib/std/rand/Ascon.zig:53:19
    remaining reference traces hidden; use '-freference-trace' to see all reference traces
`

OS - `Kubuntu 22.04.2 LTS x86_64`

Thank you for your response.
After cloning  repo again and running 
`
git submodule update --init --recursive
`
I still get 
`
zig build-exe random Debug thumb-freestanding-none: error: the following command failed with 1 compilation errors:
/snap/zig/7734/zig build-exe /home/alex/raspberrypi-rp2040/deps/microzig/src/start.zig -fno-strip --cache-dir /home/alex/raspberrypi-rp2040/zig-cache --global-cache-dir /home/alex/.cache/zig --name random -fcompiler-rt -fsingle-threaded -target thumb-freestanding-none -mcpu cortex_m0plus --script /home/alex/raspberrypi-rp2040/rp2040.ld --mod board:microzig:/home/alex/raspberrypi-rp2040/src/boards/raspberry_pi_pico.zig --mod hal:microzig:/home/alex/raspberrypi-rp2040/src/hal.zig --mod chip:microzig:/home/alex/raspberrypi-rp2040/src/chips/RP2040.zig --mod config::/home/alex/raspberrypi-rp2040/zig-cache/microzig/config-a7f1609d0013964d3f3a48cf7eef1cea.zig --mod cpu:microzig:/home/alex/raspberrypi-rp2040/deps/microzig/src/modules/cpus/cortex-m.zig --mod app:microzig:/home/alex/raspberrypi-rp2040/examples/random.zig --mod microzig:config,chip,cpu,hal,board:/home/alex/raspberrypi-rp2040/deps/microzig/src/microzig.zig --deps app,microzig --listen=- 
Build Summary: 24/27 steps succeeded; 1 failed (disable with --summary none)
install transitive failure
└─ install random transitive failure
   └─ zig build-exe random Debug thumb-freestanding-none 1 errors
/snap/zig/7734/lib/std/math.zig:437:17: error: deprecated; use @min instead
pub const min = @compileError("deprecated; use @min instead");
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
referenced by:
    min: /snap/zig/7734/lib/std/rand/Ascon.zig:12:21
    fill: /snap/zig/7734/lib/std/rand/Ascon.zig:53:19
    remaining reference traces hidden; use '-freference-trace' to see all reference traces
`

My zig version - `0.11.0-dev.3731+c6e2e1ae4`

I have reinstalled zig(0.11.0-dev.3747+7b5bd3a93). Now when I try to build I get another error:
`
deps/microzig/src/start.zig:144:25: error: invalid builtin function: '@ptrToInt'
        const bss_len = @ptrToInt(bss_end) - @ptrToInt(bss_start);
`

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -1,0 +1,22 @@
+name: Build
+on:
+  push:
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [
+          linux-latest,
+          windows-latest,
+          macos-latest,
+        ]
+    steps:
+    - uses: actions/checkout@v2
+    - uses: goto-bus-stop/setup-zig@v1.3.0
+      with:
+        version: master
+
+    - name: Build and Unit Test
+      run: zig build test -Doptimize=ReleaseSmall

--- a/build.zig
+++ b/build.zig
@@ -2,7 +2,7 @@ const std = @import("std");
 const Builder = std.build.Builder;
 const Pkg = std.build.Pkg;
 const comptimePrint = std.fmt.comptimePrint;
-const FileSource = std.build.FileSource;
+const LazyPath = std.build.LazyPath;
 
 const microzig = @import("microzig");
 
@@ -17,7 +17,7 @@ pub const BuildOptions = struct {
 
 pub const PicoExecutableOptions = struct {
     name: []const u8,
-    source_file: FileSource,
+    source_file: LazyPath,
     optimize: std.builtin.OptimizeMode = .Debug,
 };
 
@@ -52,7 +52,7 @@ pub fn build(b: *Builder) !void {
         },
         .optimize = optimize,
     });
-    pio_tests.addIncludePath("src/hal/pio/assembler");
+    pio_tests.addIncludePath(.{ .path = "src/hal/pio/assembler" });
 
     const test_step = b.step("test", "run unit tests");
     test_step.dependOn(&b.addRunArtifact(pio_tests).step);

--- a/build.zig.zon
+++ b/build.zig.zon
@@ -3,8 +3,8 @@
     .version = "0.0.0",
     .dependencies = .{
         .microzig = .{
-            .url = "https://github.com/ZigEmbeddedGroup/microzig/archive/9392fe0f7bddde26155c181ab80b70097b49c791.tar.gz",
-            .hash = "1220326148075cd017425e04356329c33c3e794d31a54bdafa46521fcfcc55b422a3",
+            .url = "https://github.com/ZigEmbeddedGroup/microzig/archive/d5c93ba33d0344bfb12415fd632577c639369b75.tar.gz",
+            .hash = "1220fd3ad4f6a88f111791cb80de1fc388d0a51114ab376f2bf8351cd176747fa303",
         },
         .args = .{
             .url = "https://github.com/MasterQ32/zig-args/archive/91d1e89fb89a4d01dec7c9aec95b0a324080ebcc.tar.gz",

--- a/build.zig.zon
+++ b/build.zig.zon
@@ -3,7 +3,7 @@
     .version = "0.0.0",
     .dependencies = .{
         .microzig = .{
-            .url = "https://github.com/ZigEmbeddedGroup/microzig/archive/d5c93ba33d0344bfb12415fd632577c639369b75.tar.gz",
+            .url = "https://github.com/ZigEmbeddedGroup/microzig/archive/d4a48f65fac24dcfe789d217a9154086b6a011e5.tar.gz",
             .hash = "1220fd3ad4f6a88f111791cb80de1fc388d0a51114ab376f2bf8351cd176747fa303",
         },
         .args = .{

